client/my-sites/media-library/content.jsx

```diff
@@ -364,6 +367,7 @@ export class MediaLibraryContent extends React.Component {
 				<MediaLibraryList
 					key="list-loading"
 					filterRequiresUpgrade={ this.props.filterRequiresUpgrade }
+					isMobile={ this.props.isMobile }
```

Comment:
If we ended up drilling the `isMobile` prop from top down to the `MediaLibraryScale` component, can we use the `mediaScale` prop instead? That's the value we _really_ want, isn't it?

That way, the `mediaScale` value will be computed only at one place, and passed down to all other components that need it.

Reply:
If we pass down `mediaScale` prop value to children components, I think they need to unsubscribe from the Redux Store because the `mediaScale` value from Redux would overwrite the value from the parent components. Would it be alright to unsubscribe?

Reply:
I'm not sure what you mean by "unsubscribe". Do you mean that the `MediaLibraryScale` preference would no longer read the `scale` prop in Redux with `getMediaScalePreference`, but it would get the value in the regular `scale` prop from the parent component? Then yes, that's alright and desirable. Ideally, there would be only one `getMediaScalePreference` call at the top of the tree and `scale` would be passed down, exactly the same way as your patch currently does with the `isMobile` prop.